src/transaction/log_prior_recv.hpp

```diff
@@ -53,13 +53,13 @@ namespace cublog
       prior_recver &operator = (prior_recver &&) = delete;
 
       void push_message (std::string &&str);                  // push message from prior_sender into message queue
+      void start_thread ();                                   // run loop_message_to_prior_info in a thread
```

Comment:
Nit
Add a remark to the class comment that this class implementation is not thread-safe because it does not need to be.